src/Makefile.in

```diff
@@ -64,18 +65,19 @@ depend::
 #
 
 install::
-	$(INSTALL) -s -m 555 lv $(bindir)
-	if test -f $(bindir)/lgrep; then \
-		/bin/rm -f $(bindir)/lgrep; \
+	if test ! -d $(DESTDIR)$(bindir); then \
+		mkdir -p $(DESTDIR)$(bindir); \
 	fi
-	(cd $(bindir); ln lv lgrep)
-	if test ! -d $(lvlibdir); then \
-		mkdir -p $(lvlibdir); \
+	$(INSTALL) -m 555 lv $(DESTDIR)$(bindir)
+	(cd $(DESTDIR)$(bindir); ln -s lv lgrep)
```

Comment:
install fails if lgrep is exists.

Reply:
Ah, good point! Restored `rm` call and tested that subsequent installs don't fail.